reports/2017-05-22.md

```diff
@@ -0,0 +1,175 @@
+# Development report for 22 May 2017
+
+The tenth weekly development report. Feel free to send PRs if you want to add to these reports (or correct them).
+
+There is now a new [linuxkit/virtsock] repository that contains Go bindings and sample code for 
+Hyper-V sockets and virtio sockets.
+
+There will be a [SIG Security call on 24 May 2017](https://github.com/linuxkit/linuxkit/blob/master/reports/sig-security/2017-05-24.md).
+
+## Overview
```

Comment:
I would add a line that we have significantly improved testing and continue adding more tests